.jenkins

```diff
@@ -462,23 +462,23 @@ pipeline {
                         }
                     }
                 }
-                stage('CUDA-11.0.3-Clang-Tidy') {
+                stage('CUDA-11.0.3-Clang-18.1.8') {
```

Comment:
I would suggest not to specify minor/patch versions as the package manager might install any (it seems you're just requiring `clang-18`, so only the major version is guaranteed to match your desires).
```suggestion
                stage('CUDA-11.0.3-Clang-18') {
```